packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -59,12 +67,6 @@ export const GRID_COL_RESIZE_STOP = 'colResizing:stop';
 
 export const GRID_COLUMN_ORDER_CHANGE = 'columnOrderChange';
 
-export const GRID_COLUMN_REORDER_START = 'columnReordering:dragStart';
```

Comment:
IMO events should be named according to the DOM events that they listen to and not the context in which they are used. Different hooks could subscribe to the same event.